easybuild/easyconfigs/p/PSI4/PSI4-1.2.1-intel-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'EB_PSI'
+
+name = 'PSI4'
+version = '1.2.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.psicode.org/'
+description = """PSI4 is an open-source suite of ab initio quantum chemistry programs designed for
+ efficient, high-accuracy simulations of a variety of molecular properties. We can routinely perform
+ computations with more than 2500 basis functions running serially or in parallel."""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://github.com/psi4/psi4/archive']
+sources = ['v%(version)s.tar.gz']
+patches = ['PSI4-%(version)s_fix_cmake_release.patch']
+checksums = [
+    'fbde7ee67174f7ba7b7f6f4e8117ce8b80f5fd10414577f5b0fcdbaa6c8ba020',  # v1.2.1.tar.gz
+    '65bdae7fa4288ab811f65165a87d4be2a4cad4fbc9bf007755d60c7a829b61e4',  # PSI4-1.2.1_fix_cmake_release.patch
+]
+
+dependencies = [
+    ('Python', '2.7.15'),
+    ('CheMPS2', '1.8.8'),
+    #  Tests are failing with pytest 3.9.2
+    ('pytest', '3.8.2', versionsuffix),
+    ('networkx', '2.2', versionsuffix),
+    ('deepdiff', '3.3.0', versionsuffix),
+]
+
+builddependencies = [
+    ('CMake', '3.12.1'),
+    # ('Perl', '5.28.0'),  # for the test suite
```

Comment:
@hajgato Should this still be commented out?

Reply:
@boegel 
Yes, see new comment:
Perl needed for the ctest test suite during the installation, not for the pytest test suite after the installation

As I explained in the easyconfig:
Runtest uses ctest, and some of the thests have to be manually compared to the reference output (those tests are marked failed After installing PSI4, you can test the package using psi4 --test command. (This uses pytest framework)
As a consequence I set `runtest` to `False`

Reply:
OK, thanks for clarifying.